components/Layout/Layout.js

```diff
@@ -2,20 +2,49 @@ import Link from "next/link";
 import Header from "../Header/Header";
 import Image from "next/image";
 import { NavBar } from "../Layout/Layout.styled";
+import { useState } from "react";
 export default function Layout({ children }) {
+  const [activeLink, setActiveLink] = useState("");
   return (
     <>
       <Header />
       {children}
       <NavBar>
-        <Link href="/">
-          <Image src="/home.svg" width={35} height={35} alt="home" />
+        <Link href="/" onClick={() => setActiveLink("home")}>
+          <Image
+            src="/home.svg"
+            width={35}
+            height={35}
+            alt="home"
+            className={activeLink === "home" ? "active" : ""}
+          />
```

Comment:
All links could then simple become:

```suggestion
        <Link href="/">
          <Image
            src="/home.svg"
            width={35}
            height={35}
            alt="home"
            className={asPath === "/" ? "active" : ""}
          />
```

Reply:
thank you for your feedback I`ll  change it now